examples/benchmark/benchmark_utils.py

```diff
@@ -0,0 +1,111 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+import time
+from dataclasses import dataclass
+
+import numpy as np
+import psutil
+import torch.cuda
+from pynvml import *
```

Comment:
It's not good practice to do star imports. Can we just import exactly what we need?